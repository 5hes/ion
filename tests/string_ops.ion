let string = world
echo $string
let string ++= !
echo $string
let string ::= "Hello, "
echo $string

let string1 = Doctor
let string2 = order
echo $string1
echo $string2
let string1 ++= "'s"
let string2 ++= "s"
echo $string1
echo $string2
let string1 ++= " "
let string2 ::= $string1
echo $string2
