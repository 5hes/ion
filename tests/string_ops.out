world
world!
Hello, world!
Doctor
order
Doctor's
orders
Doctor's orders
